src/Downloader.ts

```diff
@@ -466,9 +467,13 @@ class Downloader {
           resolve(val);
         }
       });
-      call.retryIf((err: any) => err.response && err.response.status !== 404);
+      // todo pull this logic up to the class
```

Comment:
As you have already mentioned it as Todo, I think it's better if we can create a ticket for this to note it.

Reply:
You're right, however, that's just a kind of refactoring so I don't think it deserve the ticket. I redesigned that.

Reply:
Great 👍